pom.xml

```diff
@@ -299,6 +304,12 @@
                 <version>${mockito.version}</version>
                 <scope>test</scope>
             </dependency>
+
+            <dependency>
+                <groupId>ch.qos.reload4j</groupId>
+                <artifactId>reload4j</artifactId>
+                <version>${reload4j.version}</version>
```

Comment:
what is the difference between ch.qos.reload4j.reload4j and org.slf4j.slf4j-reload4j